stdlib/3/_winapi.pyi

```diff
@@ -45,7 +45,7 @@ WAIT_OBJECT_0: int
 WAIT_TIMEOUT: int
 
 def CloseHandle(handle: int) -> None: ...
-def ConnectNamedPipe(handle: int, overlapped: Union[int, bool] = ...) -> None: ...
+def ConnectNamedPipe(handle: int, overlapped: Union[int, bool] = ...) -> int: ...
```

Comment:
Actually, it appears to be mostly correct, except for the case where you call it with overlapped. Based on https://github.com/python/cpython/blob/266f4904a222a784080e29aad0916849e507515d/Modules/_winapi.c#L393 it returns an OVERLAPPED object if it is called with overlapped. Otherwise, it returns `None`.